SupportClient/SupportClient/Utilities/Constants.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Constants.swift
+//  SupportClient
+//
+//  Created by Cameron Russell on 2/15/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+struct Constants {
+    
+    static let backgroundColor = UIColor.white
+    static let themeColor = UIColor(red: 85/255, green: 212/255, blue: 212/255, alpha: 1)
+    static let navigationTintColor = UIColor(white: 237/255, alpha: 1)
+    static let titleColor = UIColor(white: 64/255, alpha: 1)
+    static let subtitleColor = UIColor(white: 159/255, alpha: 1)
+    
+    static let robotoMedium = "Roboto-Medium"
+    static let robotoRegular = "Roboto-Regular"
+    
+}
```

Comment:
we should have different extensions for `UIColor` and `UIFont`. We should have a `UIFont+Extensions.swift` and `UIColor+Extensions.swift` file. heres an example: https://github.com/cuappdev/uplift-ios/blob/master/Uplift/Extensions/UIColor%2BShared.swift